sdks/wasm/Makefile

```diff
@@ -233,7 +246,22 @@ run-all-%:
 	make -C . run-sm-$*
 	make -C . run-jsc-$*
 
+run-browser-tests: $(BROWSER_TEST)/.stamp-browser-test-suite
```

Comment:
This target should probably be listed in ../README.md along with the others for wasm

Reply:
I will be updating the ../README.md in a separate PR.  Was waiting on some feedback.